framework/CodeInterfaces/RELAP5/Relap5Interface.py

```diff
@@ -107,16 +107,19 @@ def checkForOutputFailure(self,output,workingDir):
       @ Out, failure, bool, True if the job is failed, False otherwise
     """
     failure = True
-    errorWord = ["Transient terminated by end of time step cards","Transient terminated by trip"]
+    errorWord = ["Transient terminated by failure","Thermodynamic property error with minimum time step, transient being terminated"]
```

Comment:
If we just use the "goodWord" list now, we probably can delete the errorWord list.

Reply:
I left it there as a note to discuss if it is worth to check error word as well as a redundancy measures. It surprises me this issue did not come out in the past.